app/src/main/java/ayds/spotisong/aurora/otherdetails/model/repository/OtherInfoRepositoryImp.kt

```diff
@@ -0,0 +1,32 @@
+package ayds.spotisong.aurora.otherdetails.model.repository
+
+import ayds.spotisong.aurora.otherdetails.model.repository.external.OtherInfoExternalService
+import ayds.spotisong.aurora.otherdetails.model.repository.local.DataBase
+import ayds.spotisong.aurora.song.model.Song
+
+private const val EXISTE_EN_DB_TAG = "[*]"
+
+internal class OtherInfoRepositoryImp(val service : OtherInfoExternalService, val dataBase: DataBase) : OtherInfoRepository{
```

Comment:
definir las propiedades privadas 
```suggestion
internal class OtherInfoRepositoryImp(private val service : OtherInfoExternalService, private val dataBase: DataBase) : OtherInfoRepository{
```